jenkins_jobs/periodic_pipelines.groovy

```diff
@@ -6,7 +6,7 @@ def createPeriodicPipeline(String name) {
 node('master') {
   dir('infrastructure') {
     git(url: 'https://github.com/$GITHUB_ORGANIZATION_NAME/infrastructure.git',
-        branch: 'master')
+        branch: '$REPOSITORY_COMMIT')
```

Comment:
The same way `GITHUB_ORGANIZATION_NAME`, via the seed job parameters. Is that wrong?